src/extension/engine/AbstractServer.ts

```diff
@@ -130,7 +130,16 @@ export class AbstractServer
      * @return {string[]}
      */
     splitQueries(sqlMulti: string) {
```

Comment:
I my opinion we must move body this function to postgres class. Function should return by default sqlMulti param value.
```
splitQueries(sqlMulti: string) {
    return sqlMulti;
}
```
The function written in this way enables us to use the flag `multipleStatements` in MySQL.